website/src/pages/case-studies/nutanix-fixed-log4j-with-sourcegraph.tsx

```diff
@@ -0,0 +1,332 @@
+import * as React from 'react'
+import { Link } from 'gatsby'
+import { CaseStudyJumbotron, AuthorBio } from '../../components/content/CaseStudyPage'
+import { CaseStudiesList } from '../../components/product/CaseStudiesList'
+import { ContentSection } from '../../components/content/ContentSection'
+import Layout from '../../components/Layout'
+import ClipboardTextOutlineIcon from 'mdi-react/ClipboardTextOutlineIcon'
+import ChartBarIcon from 'mdi-react/ChartBarIcon'
+import CheckCircleOutlineIcon from 'mdi-react/CheckCircleOutlineIcon'
+import FlagOutlineIcon from 'mdi-react/FlagOutlineIcon'
+import slugify from 'slugify'
+
+export default ((props: any) => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Nutanix fixed Log4j quickly and confidently with Sourcegraph',
+            description:
+                'Sourcegraph enables Nutanix to find and fix security vulnerabilities quickly across the code base. When the Log4j vulnerability rocked the industry, Nutanix was able to confidently identify every instance of Log4j across its sprwaling codebase and deliver patches to its customers that fully remediated the vulnerability within 4 days.',
+            image: 'https://about.sourcegraph.com/sourcegraph-og.png',
+        }}
+    >
+        <div className={`${slugify('Nutanix').toLowerCase()}-case-study`}>
+            <CaseStudyJumbotron
+                className="bg-gradient-onahau-fog text-black height-md-450 height-auto p-2"
+                color="white"
+                customer="Nutanix"
+            >
+                <h1 className="pt-5 pb-6 display-2 font-weight-bold mw-600 mx-auto">
+                    Nutanix fixed Log4j quickly and confidently with Sourcegraph
+                </h1>
+            </CaseStudyJumbotron>
+
+            <ContentSection color="white" className="py-6 text-center col-sm-12 col-md-9 col-lg-7">
+                <blockquote>
+                    <div className="my-4">
+                        <img src="/external-logos/nutanix-logo.svg" width="210px" alt="nutanix" />
+                    </div>
+                    <h2 className="display-3 fw-550 mw-600 mx-auto">
+                        "Sourcegraph was the right product at the right time."
```

Comment:
I think the quotes need to be tweaked here to match the design
![Screen Shot 2022-03-31 at 11 13 00 AM](https://user-images.githubusercontent.com/43454804/161095104-457986bf-1b0a-4d27-b0d4-35132c27434e.png)

![Screen Shot 2022-03-31 at 11 41 28 AM](https://user-images.githubusercontent.com/43454804/161095233-08281c62-f68e-468d-9afe-09cc1aeab918.png)

Reply:
Yeah this seems like a good place to use the `blockquote`,`figcatpion`, and `cite` elements with the HTML entities for the quotes.